delete-review-app/action.yml

```diff
@@ -0,0 +1,140 @@
+name: Delete Review App
+description: Delete Review App
+inputs:
+  azure-credentials:
+    description: A JSON string containing service principal credentials.
+    required: true
+  gcp-wip:
+    description: The full identifier of the GCP Workload Identity Provider.
+    required: false
+  gcp-project-id:
+    description: The name of the GCP Project ID.
+    required: false
+  environment:
+    description: Environment to delete from
+    default: review
+    required: false
+  github-event-name:
+    description: github event
+    default: pull_request
+    required: false
+  global-config:
+    description: Environment for global config parameters
+    default: review
+    required: false
+  pr-number:
+    description: PR number for the review app
+    required: true
+  service-name:
+    description: name of the service
+    required: true
+  service-pretty:
+    description: pretty name of the service
+    required: true
+  service-short:
+    description: short name of the service
+    required: true
+  terraform-base:
+    description: Path to the terraform files
+    required: false
+    default: "terraform/application"
+  terraform-destroy-command:
```

Comment:
we should hardcode the destroy command, and remove as an input. It can be aliased/updated in the calling service makefile

Reply:
removed